Use GraphBuilder in unit tests for ops removal.

eigen-k opened the pull request:
Summary: Use GraphBuilder in unit tests for ops removal.

Reviewed By: zonglinpeng

Differential Revision: D75026207

zonglinpeng commented on backends/cadence/aot/tests/test_fusion_ops_passes.py:
```diff
@@ -247,6 +244,7 @@ def forward(self, x):
             count_node(graph_module, exir_ops.edge.aten.view_copy.default), 2
         )
 
+    # Already covered
```
are these commented needed?

eigen-k committed:
Use GraphBuilder in unit tests for ops removal. (#11010)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/11010

Use GraphBuilder in unit tests for ops removal.

Reviewed By: zonglinpeng

Differential Revision: D75026207
